test/system/252-quadlet.bats

```diff
@@ -1542,4 +1542,27 @@ EOF
     service_cleanup $QUADLET_SERVICE_NAME inactive
     run_podman rmi $untagged_image:latest $built_image $(pause_image)
 }
+
+@test "quadlet - output and error message" {
```

Comment:
We already check the journal in `quadlet - oneshot`, I suggest you add you stdout/err check in this test for timing reasons

Reply:
Hi @Luap99 Thanks a lot for your advice. But in the oneshot case the point it write the logs to journald is a little different. In oneshot case the log is checked after the process is finished. But in this case want to check if we can get the logs during the process is still running just like a normal services. So I'd like to keep it a separated one.

Reply:
Thank you for your PR. Can you clarify what the purpose of that testing would be? Is there some reason to believe that podman could possibly buffer journal output until a process terminates? There _is_ buffering / time delay going on, but [it is not podman's fault](https://github.com/systemd/systemd/issues/28650). And as @Luap99 mentioned, your test will flake due to this race condition.

Reply:
This case want to check the services that started by the service file generated from quadlet can send stdout/stderr/syslog to the journal. Normally the services will keep running so from the user view there maybe have a slight different with the test case with oneshot. I am not saying there is problem exist in the podman write logs to journal during the process is running, but that is something that my test scenario want to covered and checked.

Reply:
It is fine to cover that but I think it still makes more sense to just extend an existing test for that.
The quadlet tests are not exactly fast as they need to generate/daemon-reload/start unit every time so I think it would help if you just add the print to an existing unit

Reply:
I still prefer this being added to an existing test, i.e. `"quadlet - ContainerName`.
quadlet tests are slow (around 2.5s), it does not sounds like much but in the sum this adds up quickly.
As person doing a lot of work to speed system test up (https://github.com/containers/podman/pull/22826, https://github.com/containers/podman/pull/22821) we really need to be more careful when adding new tests.